canonical/ubuntu.com
# [copy update] /download/amd-xilinx

## Done

- Updated copy and added new tab as per [doc](https://docs.google.com/document/d/1cxQnEErs8FZ3dZ_IhlfcaJ03gE_vrMJfRC-5me9IDd4/edit#)
- Drive by's:
  - Divided tabs into separate files so that they're easier to maintain 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-12902.demos.haus/download/amd-xilinx
    - Be sure to test on mobile, tablet and desktop screen sizes
- See that the requested changes have been made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3924
